components/sanity/scripts/start.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+
+import os
+import json
+import requests
+from os import remove
+from os.path import join, dirname, isfile
```

Comment:
join and dirname are imported like this here just because they're only used in the utils download part. please don't use isfile directly.. as it is rather ambiguous. us os.path.isfile instead please.